### PR TITLE
feat(MMPD-546): update portfolio icon

### PR DESCRIPTION
**Pull request by GuiBibeau**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

Due to a redesign, we are changing the link to portfolio from a simple icon to instead be a secondary button. 

**Screenshots/Recordings**

Screenshot 2023-07-06 at 12 52 45 PM

Screenshot 2023-07-12 at 4 47 40 PM

How to test:

1. Go to the main view
2. click the portfolio button and make sure the portfolio link works like before

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/UI/Tokens/index.tsx [hunk 1]
```diff
@@ -459,17 +459,15 @@ const Tokens: React.FC<TokensI> = ({ tokens }) => {
...
         </Text>
-        <TouchableOpacity
+        <Button
+          variant={ButtonVariants.Secondary}
+          size={ButtonSize.Md}
+          width={ButtonWidthTypes.Full}
+          style={styles.buyButton}
           onPress={onOpenPortfolio}
-          style={styles.portfolioLink}
-          {...generateTestId(Platform, PORTFOLIO_BUTTON)}
```
Comment (sethkfman) [inline app/components/UI/Tokens/index.tsx:465]: I would make sure this ID still exists in the new component

File: app/components/UI/Tokens/index.tsx [hunk 2]
```diff
@@ -393,7 +393,7 @@ const Tokens: React.FC<TokensI> = ({ tokens }) => {
           width={ButtonWidthTypes.Full}
           style={styles.buyButton}
           onPress={goToBuy}
-          label={strings('wallet.buy_asset', { asset: mainToken.symbol })}
+          label={strings('wallet.portfolio_button')}
```
Comment (sethkfman) [inline app/components/UI/Tokens/index.tsx:396]: Is the buy button being replaced?

Comment (GuiBibeau) [reply app/components/UI/Tokens/index.tsx:396]: Oh sorry that's a mistake on my part that was supposed to go to another button. Updating.

File: app/components/UI/Tokens/index.tsx [hunk 3]
```diff
@@ -459,17 +459,15 @@ const Tokens: React.FC<TokensI> = ({ tokens }) => {
...
+        <Button
+          variant={ButtonVariants.Secondary}
+          size={ButtonSize.Md}
+          width={ButtonWidthTypes.Full}
+          style={styles.buyButton}
           onPress={onOpenPortfolio}
-          style={styles.portfolioLink}
-          {...generateTestId(Platform, PORTFOLIO_BUTTON)}
-        >
-          <Icon
```
Comment (sethkfman) [inline app/components/UI/Tokens/index.tsx:467]: I would remove these components from the import. Run `yarn lint` and confirm you don't have any errors.

**Comment by sethkfman** [comment]
@yanrong-chen Can you take a look at the UI above to make sure it is aligned with platform design?

File: app/components/UI/Tokens/index.tsx (hunk 3 above)
Comment (GuiBibeau) [reply app/components/UI/Tokens/index.tsx:467]: Good point, ran it and remove imports

File: app/components/UI/Tokens/index.tsx (hunk 1 above)
Comment (GuiBibeau) [reply app/components/UI/Tokens/index.tsx:465]: Good call, I added it to the new component too.

File: app/components/UI/Tokens/index.tsx [hunk 4]
```diff
@@ -457,17 +453,16 @@ const Tokens: React.FC<TokensI> = ({ tokens }) => {
...
+          label={strings('asset_overview.portfolio_button')}
           {...generateTestId(Platform, PORTFOLIO_BUTTON)}
-        >
-          <Icon
-            color={IconColor.Primary}
-            name={IconName.Diagram}
-            size={IconSize.Md}
-          />
-        </TouchableOpacity>
+          endIconName={IconName.Login}
```
Comment (GuiBibeau) [inline app/components/UI/Tokens/index.tsx:464]: This is the icon that might change.

**Comment by yanrong-chen** [comment]
> @yanrong-chen Can you take a look at the UI above to make sure it is aligned with platform design?

Hi Seth, we are using a different icon in this portfolio entry button
portfolio entry(1)

**Comment by jclancy93** [comment]
LGTM from the portfolio side

**Review by jclancy93 — DISMISSED** [review]

**Comment by GuiBibeau** [comment]
@yanrong-chen @sethkfman I'm currently waiting to get Figma access to have the right account to put here. I left a comment on this in the PR description.
https://github.com/MetaMask/metamask-mobile/pull/6751#issuecomment-1623975440

**Comment by yanrong-chen** [comment]
> @yanrong-chen @sethkfman I'm currently waiting to get Figma access to have the right account to put here. I left a comment on this in the PR description. [#6751 (comment)](https://github.com/MetaMask/metamask-mobile/pull/6751#issuecomment-1623975440)

I moved this design to the following link, pls let me know if you still can't access the page 
https://www.figma.com/file/fFuWPCTvJ9ZYHIxFyRq0HH/portfolio-entry?type=design&node-id=0-1&mode=design&t=ayqoKxzA7R2vFqu3-0

**Comment by GuiBibeau** [comment]
> > @yanrong-chen @sethkfman I'm currently waiting to get Figma access to have the right account to put here. I left a comment on this in the PR description. [#6751 (comment)](https://github.com/MetaMask/metamask-mobile/pull/6751#issuecomment-1623975440)
> 
> I moved this design to the following link, pls let me know if you still can't access the page https://www.figma.com/file/fFuWPCTvJ9ZYHIxFyRq0HH/portfolio-entry?type=design&node-id=0-1&mode=design&t=ayqoKxzA7R2vFqu3-0

Thanks that gave me the name! I updated the code and screenshot

**Review by jclancy93 — APPROVED** [review]

**Comment by yanrong-chen** [comment]
Hi @GuiBibeau - The new screenshot looks great to me! Thank you for the fix.

**Review by MarioAslau — APPROVED** [review]
lgtm !

**Pull request closed without merging**